### PR TITLE
right clicking swatches

### PR DESCRIPTION
**Pull request by lassiiter**
Reopening of:
https://github.com/armory3d/armorpaint/pull/1019

Implemented Feature Request / Bug:
https://github.com/armory3d/armorpaint/issues/1018

Right clicking swatches now selects them when you want to right click and delete, shows indicator over swatch right clicked

### CODE REVIEW

File: Sources/arm/ui/TabSwatches.hx [hunk 1]
```diff
@@ -103,10 +103,12 @@ class TabSwatches {
 					}
 					if (ui.isHovered && ui.inputReleasedR) {
 						var add = Project.raw.swatches.length > 1 ? 1 : 0;
+						Context.setSwatch(Project.raw.swatches[i]);
+						UISidebar.inst.hwnd2.redraws = 2;
```
Comment (luboslenco) [inline Sources/arm/ui/TabSwatches.hx:107]: Is the redraw tag needed?

File: Sources/arm/ui/TabSwatches.hx [hunk 2]
```diff
@@ -103,10 +103,12 @@ class TabSwatches {
...
 						UIMenu.draw(function(ui: Zui) {
 							ui.text(tr("Swatch"), Right, ui.t.HIGHLIGHT_COL);
 							if (Project.raw.swatches.length > 1 && ui.button(tr("Delete"), Left)) {
-								Context.setSwatch(Project.raw.swatches[i == 0 ? 1 : 0]);
+								Context.setSwatch(Project.raw.swatches[i]);
```
Comment (luboslenco) [inline Sources/arm/ui/TabSwatches.hx:111]: Why this change?

File: Sources/arm/ui/TabSwatches.hx (hunk 1 above)
Comment (lassiiter) [reply Sources/arm/ui/TabSwatches.hx]: Removed this redraw :)

File: Sources/arm/ui/TabSwatches.hx (hunk 2 above)
Comment (lassiiter) [reply Sources/arm/ui/TabSwatches.hx]: Reverted with set swatch change
